index.html

```diff
@@ -0,0 +1,237 @@
+            <li>
+              <a href="#portfolio" class="desktop-menu-item default-custom-font">
+                Portfolio
+              </a>
+            </li>
+            <li>
+              <a href="#about" class="desktop-menu-item default-custom-font">
+                About
+              </a>
+            </li>
+            <li>
+              <a href="#contact" class="desktop-menu-item default-custom-font">
+                Contact
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <section id="headline-container" class="headline-container flex-container-column">
+      <div class="headline primary-text flex-container-column">
+        <h1 class="primary-text first-color primary-text-custom-font">Hey there. I'm Habtamu Alemayehu</h1>
+        <h2 class="primary-text second-color primary-text-custom-font">I'm a software developer</h2>
+      </div>
+      <div class="headline supporting-text">
+        <p class="sup-text-paragraph default-custom-font">I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+      </div>
+      <ul class="headline social-media-icons align-center">
+        <li class="headline-item">
+          <a class="twitter" href="https://twitter.com/MandeAlemayehu" title="Contact-me: Twitter link"><img src="./css/img/headline/icons/twitter.png" alt="My twitter"></a>
+        </li>
+        <li class="headline-item">
+          <a class="linkedin" href="https://www.linkedin.com/in/habtamu-alemayehu-b90367101/" alt="My linkedin"><img src="./css/img/headline/icons/linkedin.png"/></a>
+        </li>
+        <li class="headline-item">
+          <a class="medium" href="https://medium.com/@habtamualemayehu16" title="Contact-me: Medium link"><img src="./css/img/headline/icons/medium.png" alt="My medium"></a>
+        </li>
+        <li class="headline-item">
+          <a class="github" href="https://github.com/Benawi" title="Contact-me: Github link"><img src="./css/img/headline/icons/github.png" alt="My github"></a>
+        </li>
+        <li class="headline-item">
+          <a class="angellist" href="https://venture.angellist.com/v/login" title="Contact-me: AngelList link"><img src="./css/img/headline/icons/angellist.png" alt=""></a>
+        </li>
+      </ul>
+      <div class="headline action">
+        <a class="action-btn" href="#" title="Display next section">
+          <div class="chevron"></div>
+        </a>
+      </div>
+    </section>
+    <section id="about" class="about-me-container flex-container-column">
+      <div class="about-me text flex-container-column">
+        <div class="about-me header flex-container-row align-center">
+          <h2 class="about-primary-text primary-text first-color primary-text-custom-font">
+            About me
```

Comment:
- Great job implementing this "Finish mobile version" of your portfolio 👍🏽 
- I think you may have forgotten to implement the project cards (for your case, "My recent works"). Can we implement the project container section that consists of 6 project cards? 

Here is what you have compared to Figma👇🏽 

| Recent Works missing in your design | Figma Design |
| :---:   | :---: | 
| <img width="394" alt="Screenshot 2023-03-24 at 08 52 55" src="https://user-images.githubusercontent.com/99246359/227542957-e82f3942-60a2-4610-9d48-71fcc54b9373.png"> |<img width="279" alt="Screenshot 2023-03-24 at 09 04 22" src="https://user-images.githubusercontent.com/99246359/227543052-946fecdc-85a1-4f4e-ae3b-b2f7dfa58a5d.png">  |